packages/expo-speech/CHANGELOG.md

```diff
@@ -10,6 +10,12 @@
 
 ### 💡 Others
 
+## 11.1.0 — 2023-01-5
+
+### 🎉 New features
+
+- Added utterance word tracking support for iOS and Android. This allows the ability to highlight each word in an utterance. ([#20726](https://github.com/expo/expo/pull/20726) by [@gabrieljoelc](https://github.com/gabrieljoelc))
+
```

Comment:
This should be in the `## Unpublished` section

Reply:
If you have any ideas on how to make the bot suggestion less confusing, let us know 😉 

cc @tsapeta